### ISSUE TITLE
Discussing+Considering card keeping/collection as an alternate form of gameplay.

### ISSUE THREAD

**Issue by SkylarkHYP** [db0/hypnagonia#356]
I had an idea in mind for a radical mechanic that I just wanted to blurt out.
@Azure-Blaze recently mentioned a "bag of holding" curio, which reminded me a lot of the popular CCG's that I used to play. (Like on kongregate and other stuff)

Basically having a separate deck and player inventory would greatly help hypnagonia's branch off from StS gameplaywise, but ultimately sounds too farfetched from the original premise, that's why I see it as an alternate form, rather than apart of the core gameply. 

The problem is that I've never heard of or played a "roguelike CCG. But the alteration to gameplay as a second difficulty or gamemode would be interesting. 

To summarize, the ability to keep or collect cards during a run would be an interrsting twist to hypnagonia's gameplay, as well as seperate it from the StS mold, but would be too game-changing. Therefore, this is only a suggestion, not a push.

Let me know your opinion on my "bag of holding" gamemode 🤣
